packages/token-lists/src/top-100.ts

```diff
@@ -0,0 +1,124 @@
+import fs from "fs";
+import path from "path";
+import { request, gql } from "graphql-request";
+import { getAddress } from "@ethersproject/address";
+
+// Interface for Bitquery GraphQL response.
+interface BitqueryEntity {
+  Total_USD: number;
+  baseCurrency: {
+    address: string;
+    name: string;
+    symbol: string;
+    decimals: number;
+  };
+}
+
+// Default token list for exchange + manual exclusion of broken BEP-20 token(s)
+const blacklist: string[] = [
```

Comment:
Simple 'blacklist' to remove default tokens already included in PancakeSwap exchange default list; plus broken/unverified BEP-20 tokens.